### PR TITLE
Compatibility With NodeBB v1.11.0. 

### PR DESCRIPTION
**Pull request by greenhat616**
To dismiss warning, to do :
* Use require.main.require instead of module.parent.require  (recommended by Node.js Docs & NodeBB Warning)

### CODE REVIEW

**Comment by greenhat616** [comment]
😃  @Dravere

**Comment by Dravere** [comment]
Yes, I know, I saw it. Had no time to test it so far.

File: library.js [hunk 1]
```diff
@@ -2,18 +2,18 @@
 
 /* globals module, require */
 
-const db = module.parent.require('./database');
-const winston = module.parent.require('winston');
+const db = require.main.require('./src/database');
+const winston = require.main.require('winston');
 const engine = require('solr-client');
-const async = module.parent.require('async');
+const async = require.main.require('async');
```
Comment (julianlam) [inline library.js]: Might be better to install async locally, and then `require('async');`

**Comment by julianlam** [comment]
LGTM, even the async thing is optional.

**Comment by Dravere** [comment]
LGTM as well. Will merge it like that. Then adapt the async dependency and release it as a bug fix.

**Pull request merged**